# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/texture_cache.rs
# Commit: c09a51622e98
# Full Hash: c09a51622e988e81077c75e3aaa5a49df66284b3
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2019-03-22 16:44:08
# Regressor Bug: 1441308
# File Overlap Count: 2
# Description:
#   Bug 1441308 - Core renderroot splitting changes r=kats,sotaro
#   
#   This is a large patch that contains all of the core changes for
#   renderroot splitting.
#   
# ==============================================================================

diff -r 737807563dd5 -r c09a51622e98 gfx/wr/webrender/src/texture_cache.rs
--- a/gfx/wr/webrender/src/texture_cache.rs	Thu Mar 21 23:14:26 2019 +0000
+++ b/gfx/wr/webrender/src/texture_cache.rs	Fri Mar 22 03:15:14 2019 +0000
@@ -581,7 +581,7 @@
     pub fn new_for_testing(max_texture_size: i32, max_texture_layers: usize) -> Self {
         let tile_size = DeviceIntSize::new(64, 64);
         let mut cache = Self::new(max_texture_size, max_texture_layers, tile_size);
-        let mut now = FrameStamp::first(DocumentId(IdNamespace(1), 1));
+        let mut now = FrameStamp::first(DocumentId::new(IdNamespace(1), 1));
         now.advance();
         cache.begin_frame(now);
         cache